src/middlewared/middlewared/plugins/smb.py

```diff
@@ -265,14 +321,15 @@ def getparm(self, parm, section):
         Str('unixcharset'),
         Str('loglevel', enum=['NONE', 'MINIMUM', 'NORMAL', 'FULL', 'DEBUG']),
         Bool('syslog'),
+        Bool('aapl_extensions'),
         Bool('localmaster'),
         Str('guest'),
         Str('admin_group', required=False, default=None, null=True),
         Str('filemask'),
         Str('dirmask'),
         Bool('ntlmv1_auth'),
         List('bindip', items=[IPAddr('ip')], default=[]),
-        Str('smb_options', max_length=None),
+        List('smb_options', default=[]),
```

Comment:
Curious what you all (@freenas/middleware) think about this. How should we handle backward compatibility of the API? Should we care?

Reply:
Whoops forgot to undo that one. I was considering switching to a list because I hated all the \n characters.